zk/stages/stage_batches.go

```diff
@@ -177,6 +178,10 @@ func SpawnStageBatches(
 				if err != nil {
 					return fmt.Errorf("write fork id error: %v", err)
 				}
+				// TODO Optimize Multiple
```

Comment:
it is unclear, what does this TODO mean, please write it more fully

Reply:
Done